cumulus/polkadot-parachain/src/cli.rs

```diff
@@ -63,9 +63,9 @@ pub enum Subcommand {
 
 const AFTER_HELP_EXAMPLE: &str = color_print::cstr!(
 	r#"<bold><underline>Examples:</></>
-   <bold>polkadot-parachain --chain asset-hub-polkadot --sync warp -- --chain polkadot --sync warp</>
+   <bold>polkadot-parachain --chain asset-hub-westend --sync warp -- --chain polkadot --sync warp</>
```

Comment:
This command with `polkadot` should work. If not, it's a breaking change for people running nodes. We'll need docs about how to migrate.